### ISSUE TITLE
Remove 1st borrow option from IA sponsorship & fix meta tags

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#3887]
This is no  longer applicable.

**Comment by mekarpeles** [comment]
Also, this issue is not complete until we also change the metadata on the archive.org items (in petabox) so they don't redrow.

>mek: Have you ever seen a case where a field in ItemDropper gets inserted w/ an extra xml closing tag?

>> 11:22 elizabethmacleod  [10:45 AM]
```
Hiya, Mek! I don't know why, but this book sponsorship item has one blank contributor and one correct contributor, which caused the billing task to redrow because it was not an exact match.
<contributor/>
<contributor>Internet Archive</contributor>
from meta.xml https://ia601502.us.archive.org/33/items/isbn_9780333333181/isbn_9780333333181_meta.xml
```

> mek  [11:18 AM] Whoa, why is that
> https://git.archive.org/ia/petabox/-/blob/master/www/common/Donate/BookSponsorship.inc#L43-60 (edited) 

> mek  11:34 AM In the above example <contributor/> appears to be the only tag in the whole document which has the form />

@traceypooh saves the day...

> traceypooh:rocket:  1:40 PM yes, that's definitely possible something in the chain is single-tagging that (obvi valid XML...) we've generally tried to remove blank fields from _meta.xml...

> mek  2:01 PM Is there a way to prune blank fields prior to submitting an ItemDrop?

> traceypooh:rocket:  2:19 PM sure!  if you have a value that is '' then omit it from the _meta.xml file (and/or key/val array) being submitted

> mek  2:25 PM Hm, I'm confused how it may be happening in this case because we're passing the field only once in
https://git.archive.org/ia/petabox/-/blob/master/www/common/Donate/BookSponsorship.inc#L43-60 e.g. https://catalogd.archive.org/log/1942435731. Somehow contributor","value":["","Internet Archive"]} is an array?

> traceypooh:rocket:  2:30 PM maybe all the values are supposed to be arrays and you passed in singletons?
(been awhile since i been in itemdroper...)

> mek  2:32 PM I see, so maybe contributor just needs to be an array.

> traceypooh:rocket:  2:33 PM yah, and maybe all of them

mek  2:36 PM
Interesting it looks like scanningcenter never even made it in as a value :confused:

> traceypooh:rocket:  2:51 PM yah, might be necessary to be arrays... yes, need to each be arrays so, for example:
```
      'scanningcenter' => [ self::$scanningcenter ],
```

**Comment by mekarpeles** [comment]
These two issues should be resolved by the petabox change staged here:
https://git.archive.org/ia/petabox/-/merge_requests/2427
https://git.archive.org/ia/petabox/-/commit/4ae9203d266711da5dc462331d21c01844dc13fc

**Review App**
http://review-2690.archive.org/donate/?campaign=pilot&type=sponsorship&context=ol&isbn=9789625933924 test with 4000000000000002

**Comment by mekarpeles** [comment]
This can be closed now that #4398 is merged. There's a PR on petabox side that can also be merged.
